tools/Test/SmokeTest/RmCoreSmokeTests.ps1

```diff
@@ -162,36 +162,35 @@ $resourceTestCommands = @(
     @{Name = "Az.Websites";                   Command = {Get-AzWebApp -ResourceGroupName $resourceGroupName}}
 )
 
-# $generalCommands = @(
-#     @{
-#         Name = "Import Az.Accounts in Parallel";
-#         Command = {
-#             if ($null -ne $env:SYSTEM_DEFINITIONID) {
-#                 Write-Host "Skipping because 'Start-Job' is not supported by design in scenarios where PowerShell is being hosted in other applications."
-#                 return
-#             }
-#             $importJobs = @()
-#             1..10 | ForEach-Object {
-#                 $importJobs += Start-Job -name "import-no.$_" -ScriptBlock { Import-Module Az.Accounts; Get-AzConfig; }
-#             }
-#             $importJobs | Wait-Job
-#             $importJobs | Receive-Job
-#             $importJobs | ForEach-Object {
-#                 if ("Completed" -ne $_.State) {
-#                     throw "Some jobs have failed."
-#                 }
-#             }
-#         };
-#         Retry = 0; # no need to retry
-#     }
-# )
+$generalCommands = @(
+    @{
+        Name = "Import Az.Accounts in Parallel";
+        Command = {
+            if ($null -ne $env:SYSTEM_DEFINITIONID -or $null -ne $env:Release_DefinitionId -or $null -ne $env:AZUREPS_HOST_ENVIRONMENT) {
+                Write-Warning "Skipping because 'Start-Job' is not supported by design in scenarios where PowerShell is being hosted in other applications."
+                return
+            }
+            $importJobs = @()
+            1..10 | ForEach-Object {
+                $importJobs += Start-Job -name "import-no.$_" -ScriptBlock { Import-Module Az.Accounts; Get-AzConfig; }
```

Comment:
Why the tests for AzConfig need to be added in smoke tests?

Reply:
This case is not to test AzConfig, but importing Az.Accounts in parallel (by Start-Job). The purpose of `Get-AzConfig` is to double check that Az.Accounts has been imported correctly.

As for why putting it in smoke test, it's because "Start-Job" cannot run in hosted PowerShell environment (like CI), which means I can't write it in a regular test case.